prefs.js

```diff
@@ -179,6 +183,17 @@ const App = new Lang.Class({
             hexpand: true,
             halign: Gtk.Align.START
         });
+        let saveImageLabel = new Gtk.Label({
+            label: _("Save images on history"),
+            hexpand: true,
+            halign: Gtk.Align.START
+        });
+        // need while this MR is pending: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1812
+        let pngImageLabel = new Gtk.Label({
```

Comment:
That PR was merged, is the workaround still necessary? (Assuming people update to the latest mutter.)